sections/roles.md

```diff
@@ -0,0 +1,44 @@
+Roles
+=========
+
+Roles currently must be created and assigned through the UI.
+
+Endpoints:
+
+- [Get Roles](#get-roles)
+- [Get a Role](#get-a-role)
+
+
+Get Roles
+-------------
+
+* `GET /api/v1/roles` will return a [paginated list](/README.md#pagination) of Roles.
+
+###### Example JSON Response
+
+```json
+[
+  {
+    "id": 1,
+    "value": "Senior"
+  },
+  {
+    "id": 2,
+    "value": "Junior"
+  }
+]
+```
+
+Get a Role
+-------------
+
+* `GET /api/v1/roles/1` will return the role with the ID of `1`.
+
+###### Example JSON Response
+
+```json
+{
+  "id": 1,
+  "value": "Senior"
+}
+```
```

Comment:
It'd also be good to put the `FilterChoices` API in the readme base and link to it from here, because you can do like `GET /api/v1/roles?filter_field=updated_at&filter_list=2016-01-01,2016-12-31`.